pumpkin/src/command/commands/cmd_give.rs

```diff
@@ -17,7 +17,7 @@ const DESCRIPTION: &str = "Give items to player(s).";
 
 const ARG_ITEM: &str = "item";
 
-static ITEM_COUNT_CONSUMER: BoundedNumArgumentConsumer<u32> =
+static ITEM_COUNT_CONSUMER: BoundedNumArgumentConsumer<i32> =
```

Comment:
Why should the item count be negative ?

Reply:
because minecraft does not have a parser for unsigned ints because java doe not have unisgned ints

I apparently forgot to add a min bound tho when I changed it so I'll do that rq